### PR TITLE
Nerfs Healing Viruses

### PR DESCRIPTION
**Pull request by ghost**
fuck you triple stacking virology and making medbay obsolete

fuck you your ease of getting 'im literally god unless i get revolvered' viruses

fuck you and everything, virology, DIE.

makes toxin filter less effective at treating toxin simply because triple stacking viruses is dumb and needs to die
#### Changelog

:cl:
rscadd: Sol Gov virologists are reporting radical mutations in viruses displaying the 'toxin filter' syndrome, becoming vastly less effective at handling toxins.
/:cl:

### CODE REVIEW

**Comment by BOBsHERE** [comment]
No? Stop nerfing everything.

**Comment by ghost** [comment]
no

**Comment by Kmc2000** [comment]
.....why

**Comment by ghost** [comment]
it only nerfed the healing

**Comment by BOBsHERE** [comment]
The healing is pathetic as it is.

**Comment by ghost** [comment]
stacked, it's not

aka 3 healing viruses = 3 toxin heal per tick, for infinity as long as you have the virus

if you get toxin compensation, that's basically having a weaker silver sulf and stypic powder in your system for the entire round, meaning that while comparatively it's nowhere near as effective as say, slapping a brute aptch on, you simply won't need brute patches because you can heal out of almost any situation if left alone

give this to a phytosian or podman and it's quite nightmarish

**Comment by BOBsHERE** [comment]
I play a fuck ton of virology and i stack my viruses. They are still weak as shit. Wont help you in a battle, and who cares otherwise? Mostly they help with random damage you take from atmos.

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -18,10 +18,10 @@ Bonus
...
 	name = "Toxic Filter"
-	stealth = 1
-	resistance = -4
-	stage_speed = -4
-	transmittable = -4
+	stealth = -6
+	resistance = -6
+	stage_speed = -6
+	transmittable = -6
 	level = 6
```
Comment (ShadowDeath6) [inline code/datums/diseases/advance/symptoms/heal.dm:25]: are these !!!undocumented changes!!! I see?

**Comment by Cruix** [comment]
Maybe a better solution would be to make the same symptoms from multiple viruses not stack.

**Comment by ShadowDeath6** [comment]
Eh, I'd kind of like to actually see the toxins in your system when TC and TF work together. Having magic tricord in your system is pretty stupid, anyway. Another slightly less assmad way of doing this is making multiple of the same symptom not stack, but w/e

**Comment by ghost** [comment]
that'd require effort, though

**Comment by ghost** [comment]
Adam, why don't you move to baystation? Most of your PRs are philosophies they embrace in bay.
 I'm getting tired of people who say they like our server but then want to change everything.

**Comment by ghost** [comment]
what exactly do you mean by 'philosophies they embrace in bay'

**Comment by killerx09** [comment]
Reminder that the formula for Toxic Filter is SQRT(20+X)*(2+N), where X equals the total stage speed of the custom virus, and N equals a positive number less than equal to 1.

**Pull request closed without merging**